intellij/org.eclipse.xtend.idea/src/org/eclipse/xtend/core/idea/completion/XtendCompletionContributor.xtend

```diff
@@ -58,7 +56,7 @@ class XtendCompletionContributor extends AbstractXtendCompletionContributor {
 	protected def completeAbstractSuperMethods() {
 		extend(
 			CompletionType.BASIC,
-			psiElement.withEReference(XtendPackage.Literals.XTEND_TYPE_DECLARATION__MEMBERS)
+			grammarAccess.memberAccess.newKeyword_2_2_2
```

Comment:
A little hacky - would it work to pass the EStructuralFeature XtendType.members here, instead?